src/mg/PAMI/Client/Impl/ClientImpl.php

```diff
@@ -185,9 +189,7 @@ public function open()
 	    @stream_set_blocking($this->_socket, 0);
 	    $this->_currentProcessingMessage = '';
 	    //register_tick_function(array($this, 'process'));
-	    if ($this->_logger->isDebugEnabled()) {
```

Comment:
https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-3-logger-interface.md#12-message

Using the context effectively avoids the concatenation (provided that the logger implementation is smart enough to replace the placeholders only after checking the log level)